{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "dfe6c11e_b88e4e20",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 1002325
      },
      "writtenOn": "2025-07-24T09:07:10Z",
      "side": 1,
      "message": "CI failure is unrelated to this patch.",
      "revId": "6f53229d3c1a90587e2ba31ed0fd108a78b99e2b",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}